dbms/src/Storages/DeltaMerge/DeltaMergeStore.cpp

```diff
@@ -1776,5 +1773,20 @@ SegmentReadTasks DeltaMergeStore::getReadTasksByRanges(
     return tasks;
 }
 
+String DeltaMergeStore::getLogTracingId(const DMContext & dm_ctx)
+{
+    if (!dm_ctx.tracing_id.empty())
+    {
+        return dm_ctx.tracing_id;
+    }
+    else if (const auto * dag_ctx = dm_ctx.db_context.getDAGContext(); dag_ctx != nullptr && dag_ctx->isMPPTask())
+    {
+        return fmt::format("{} Table<{}>", dag_ctx->getMPPTaskId().toString(), physical_table_id);
+    }
+    else
+    {
+        return fmt::format("Table<{}>", physical_table_id);
+    }
+}
```

Comment:
the `tracing_id` is set here:
https://github.com/pingcap/tiflash/blob/2f215f57a12c819d73528534c08098028160f49a/dbms/src/Flash/Coprocessor/DAGStorageInterpreter.cpp#L660
https://github.com/pingcap/tiflash/blob/2f215f57a12c819d73528534c08098028160f49a/dbms/src/Flash/Coprocessor/DAGStorageInterpreter.cpp#L293

If we change the code in DAGStorageInterpreter, the string defined in L1782 and L1785 will become incorrect. Maybe simplify it as follow is OK? /cc @windtalker 
```suggestion
String DeltaMergeStore::getLogTracingId(const DMContext & dm_ctx)
{
    if (!dm_ctx.tracing_id.empty())
    {
        return dm_ctx.tracing_id;
    }
    else
    {
        return fmt::format("Table<{}>", physical_table_id);
    }
}
```

Reply:
lgtm